ui-v2/tests/integration/components/templated-anchor-test.js

```diff
@@ -81,6 +83,16 @@ test('it renders', function(assert) {
       },
       result: 'http://localhost/?=',
     },
+    {
+      href: 'http://localhost/?={{deep.Name}}',
+      vars: {
+        deep: {
+          Name: '#Na/me',
```

Comment:
are there more characters we'd want to have in there as a test?

Reply:
We could test every non-url safe character, but we know that `encodeURIComponent` 'works' as its native, so as long as we know 1 character is encoded, we know all of them are.

If we were to test every character, we'd be testing `encodeURIComponent` itself, which we don't own.

On the otherhand, I think if we knew of any specific users using specific non url safe characters, that might be worth it just for visibility, but the only one I know of is `/`s. If you think any other specific ones are worthwhile I can add them in?